OWNERS_ALIASES

```diff
@@ -35,3 +35,8 @@ aliases:
     - russellb
     - hardys
     - stbenjam
+  ovirt-approvers:
+    - rgolangh
```

Comment:
Personally, I think it's best if we get the initial set of code merged  for this platform..
and then discuss the approval permissions in a separate PR.

you can keep the reviewers list in this PR.

Reply:
good, removing